web/src/app/forms/FormField.js

```diff
@@ -22,6 +22,10 @@ export class FormField extends React.PureComponent {
     // passed to the parent form's state.
     mapOnChangeValue: p.func,
 
+    // if passed, this will be used instead of the onChange from FormContainer
+    // use with care
+    overrideOnChange: p.func,
```

Comment:
We should discuss an alternative to this -- more context in a later comment (where it's being used).